### PR TITLE
add egp5

### PR DESCRIPTION
**Pull request by GregTheGreek**

### CODE REVIEW

File: egps/egp-5/egp-5.ts [hunk 1]
```diff
@@ -39,21 +41,35 @@ async function proposal() {
...
    */
 
+  // Deploy the airdrop contract
+  const airdropDeployer = new ethers.ContractFactory(
+    airdropData.abi,
+    airdropData.bytecode,
+    signer
+  );
+  const airdropContract = await airdropDeployer.deploy(
+    signer,
```
Comment (mcclurejt) [inline egps/egp-5/egp-5.ts]: it looks like you're setting the [gov address for reclaim after expiration](https://github.com/element-fi/council/blob/main/contracts/features/Airdrop.sol#L20) to your personal wallet.... this should probably be CoreVoting or the Timelock.

File: egps/egp-5/egp-5.ts [hunk 2]
```diff
@@ -39,21 +41,35 @@ async function proposal() {
...
+    hexRoot,
+    "0x5c6D51ecBA4D8E4F20373e3ce96a62342B125D6d", // ELFI Contract address
+    expiration,
```
Comment (mcclurejt) [inline egps/egp-5/egp-5.ts]: not entirely sure how ethers handles parsing date objects to uint256, maybe change this to `expiration.getTime()` to be safe

File: egps/egp-5/egp-5.ts [hunk 3]
```diff
@@ -4,13 +4,17 @@ import { LedgerSigner } from "@ethersproject/hardware-wallets";
 // Artifacts
 import timelockData from "../../artifacts/council/contracts/features/Timelock.sol/Timelock.json";
 import treasuryData from "../../artifacts/council/contracts/features/Treasury.sol/Treasury.json";
-import coreVotingData from "../../artifacts/council/contracts/CoreVoting.sol/CoreVoting.json";
+import coreVotingData from "../../artifacts/council/contracts/CoreVoting.sol/CoreVoting.json"
```
Comment (mcclurejt) [inline egps/egp-5/egp-5.ts]: minor nit but these artifact paths are no longer getting generated from `yarn run build`, we'll instead need to source them from the submodules' artifact directories going forward.

File: egps/egp-5/egp-5.ts (hunk 1 above)
Comment (GregTheGreek) [reply egps/egp-5/egp-5.ts]: Left that in there for you to catch 🙄

**Comment by GregTheGreek** [comment]
@mcclurejt addressed your comments

**Review by mcclurejt — APPROVED** [review]

**Pull request merged**